packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -187,6 +189,31 @@ export type RaisedHandState = {
   raisedHandOrderPosition: number;
 };
 
+/* @conditional-compile-remove(reaction) */
+/**
+ * State only version of {@link @azure/communication-calling#ReactionCallFeature}. {@link StatefulCallClient} will
+ * automatically listen for reaction on the call and update the state exposed by {@link StatefulCallClient} accordingly.
+ * 
+ * @public
+ */
+export interface ReactionCallFeatureState {
+  isEnable: boolean;
```

Comment:
Is this just to enable it at the composite/component level? If so leave this property out of the stateful client. (Per other comment; The stateful layer isn't tied to rendering logic and is just a reflection of the active call.)

Reply:
If this can be removed, we should be able to remove the ReactionCallFeatureState interface and just have reaction: ReactionState directly on the call object